components.json

```diff
@@ -1346,6 +1346,14 @@
 			"require": "markup",
 			"owner": "Golmote"
 		},
+		"wolfram": {
+			"title": "Wolfram language",
+			"alias": "wl",
+			"aliasTitles": {
+				"wl": "Mathematica"
```

Comment:
Wouldn't it be better to have another alias `mathematica` for this alias title?

Reply:
@RunDevelopment Ah I thought it had to do with the file extension... sure that makes sense. Is there a way to tell the prism system which extensions a language uses?

Reply:
We do file extensions via aliases too. You can have one alias `wl` and one `mathematica`.

Reply:
Awesome, I'll add that.